src/utils/validUrl.ts

```diff
@@ -27,5 +27,9 @@ function isUrlCurieFormat(str: string) {
   return curiePattern.test(str);
 }
 
-export { easyValidURL, isUrlCurieFormat };
+function externalLink(url: string): boolean {
+  return !url.startsWith('https://') && !url.startsWith('http://');
```

Comment:
I thought an external link is something that is outside of bbp's domain (so, if I click it I would be redirected to a page outside of the fusion application). For example "https://google.com". Is that not the case?